content/rest/channel-status.textile

```diff
@@ -18,62 +15,19 @@ jump_to:
-
-In order to grant permission to a user to access a "metachannel":#metachannels, however, the resource name(s) in the @capability@ must include the @[meta]@ qualifier explicitly. If you are using an API Key, you can "set up capabilities in your dashboard":https://support.ably.com/solution/articles/3000073773-is-it-possible-to-restrict-which-channels-or-permissions-an-api-key-has. If you are making use of "tokens":/core-features/authentication#token-authentication, you specify it within the token. The following are examples of capabilities that will validly permit access to a metachannel:
-
-minimize. See an example of a capability allowing subscription to all metachannels
-  ```[json]
-  {"[meta]*":["subscribe"]}
-  ```
-
-  The above will allow for the key to @subscribe@ to any @meta@ channel. The wildcard @*@ indicates anything can follow the @[meta]@ claim, so an example of a valid channel would be @[meta]log@. However, this capability will not allow for any other actions to be performed on the metachannels, nor will it allow the key to do anything with any non-metachannels.
-
-minimize. See an example of a capability allowing all permissible actions on all metachannels and all regular channels
-  ```[json]
-  {
-    "[meta]*":["*"],
-    "*":["*"]
-  }
-  ```
-
-  The above permission provides two capabilities: the ability to perform any action on any metachannel (such as @[meta]log@) with @"[meta]*":["*"]@, and the ability to perform any action on any channel (such as @another:channel@) with @"*":["*"]@. However, you are *never* able to publish or be present in a metachannel, thus this permission in effect would result in an actual permission excluding publish and presence capabilities in @[meta]@ channels due to the "intersecting capabilities":/core-features/authentication#capabilities-explained-intersection.
-
-If @[meta]@ is not specified in the permissions, you will be unable to access the metachannels however. An example of this would be the following:
-
-minimize. See an example of a capability which is unable to use metachannels
-  ```[json]
-  {
-    "*":["*"]
-  }
-  ```
-
-  Although the above provide all capabilities in all regular channel, without a @[meta]@ permission being explicitly specified, you will be unable to perform any actions on a @[meta]@ channel.
+Note that since the metadata of various channels is prone to change very frequently, unless you have a special use case within your app, Ably recommends you subscribe to the realtime events using "metchannels":/realtime/metachannels or "inband channel occupancy":"/realtine/inband-occupancy" rather than polling for status updates via REST, as this is inefficient and data is likely to become stale as soon as you have received it.
```

Comment:
> "inband channel occupancy":"/realtine/inband-occupancy"

realtine->realtime; also the path shouldn't be quoted

Reply:
Fixed.